chapter01/Intro_example_short.ipynb

```diff
@@ -0,0 +1,671 @@
+    "\n",
+    "## <font color=#CD853F> Contents of this notebook: </font>\n",
+    "  * [0. Learning objectives](#r-learning-objectives)\n",
+    "  * [1. Defining the BIP](#r-defining-the-bip)\n",
+    "      * [1.1. The forward model](#r-the-forward-model)\n",
+    "      * [1.2. The prior](#r-the-prior)\n",
+    "      * [1.3. The noise distribution](#r-the-noise-distribution)\n",
+    "      * [1.4. The data distribution](#r-the-data-distribution)\n",
+    "      * [1.5. The likelihood function](#r-the-likelihood-function)\n",
+    "      * [1.6. Putting it all together, the BIP](#r-putting-it-all-together-the-bip)\n",
+    "\n",
+    "  * [2. Solving the BIP](#r-solving-the-bip)\n",
+    "      * [2.2. Maximum a posteriori (MAP) estimate](#r-maximum-a-posteriori-map-estimate)\n",
+    "      * [2.3. Sampling from the posterior](#r-sampling-from-the-posterior)\n",
+    "\n",
+    "  * [3. Summary](#r-summary)\n",
+    "\n",
+    "  * [References](#r-references)\n",
+    "\n",
+    "\n",
+    "\n",
+    "## <font color=#CD853F> 0. Learning objectives </font> <a name=\"r-learning-objectives\"></a>\n",
+    "  * Create a simple BIP in CUQIpy\n",
+    "      * Create a linear forward model object\n",
+    "      * Create distribution objects that represent the prior, the noise, and the data distributions\n",
+    "      * Use the `BayesianProblem` class to define the BIP\n",
+    "\n",
+    "  * Solve a simple BIP in CUQIpy\n",
+    "      * Compute the maximum a posteriori (MAP) estimate\n",
+    "      * Sample from a simple posterior distribution and visualize the results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## <font color=#CD853F> 1. Defining the BIP </font> <a name=\"r-defining-the-bip\"></a>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Consider the following inverse problem: given observed data $b$, determine $x_1$, and $x_2$:\n",
+    "\n",
+    "$$b = x_1 + x_2 + e \\;\\;\\mathrm{with}\\;\\; e \\sim \\mathrm{Gaussain}(0, 0.1)$$\n",
+    "\n",
+    "We can write it as:\n",
+    "\n",
+    "$$b = \\mathbf{A}\\mathbf{x} + e = \\large(1,1\\large)\\binom{x_1}{x_2} + e$$\n",
+    "\n",
+    "\n",
+    "|variable     |    description          |dimension      |\n",
+    "|:------------|:------------------------|:--------------|\n",
+    "|$\\mathbf{x}$ |parameter to be inferred |2-dimensional​  |\n",
+    "|$\\mathbf{A}$ |forward model            |1-by-2 matrix  |\n",
+    "|$b$          |data                     | 1-dimensional |\n",
+    "|$e$          |noise                    |1-dimensional  ​|\n",
+    "\n",
+    "This problem is:\n",
+    "* Considered a Linear inverse problem since the forward model is linear.\n",
+    "* Ill-posed (in the sense of Hadamard [1]) since the solution is not unique, i.e., for some given value of $b$, e.g., $b=3$, all points $(x_1, x_2)$ that satisfy $x_1 + x_1 = 3$ are solutions to the (noise-free) problem."
```

Comment:
```suggestion
    "* Ill-posed (in the sense of Hadamard [1]) since the solution is not unique, i.e., for some given value of $b$, e.g., $b=3$, all parameter pairs $(x_1, x_2)$ satisfying $x_1 + x_1 = 3$ are solutions to the (noise-free) problem."
```